pkg/agent/run.go

```diff
@@ -176,27 +178,35 @@ func gatherAndPostData(ctx context.Context) {
 	log.Println("Running Agent...")
 	log.Println("Posting data to ", baseURL)
 	if config.OrganizationID == "" {
-		data, err := json.Marshal(readings)
+		data, err := json.MarshalIndent(readings, " ", " ")
```

Comment:
Ideally, we only indent the json when we're writing to disk. Data sent over the wire should be as small as possible.